2022/incoming/the-christmas-time-machine.pod

```diff
@@ -0,0 +1,81 @@
+Author: Thibault DUPONCHELLE <thibault.duponchelle@gmail.com>
+Title: The Christmas time machine
+Topic: Feature::Compat::Try
+
+=encoding utf8
+
+=head1 The Christmas time machine
+
+=head2 Terrible flashback at the line work
+
+At North Pole, elves are just starting the Christmas 2022 preparations.
+
+As usual they restarted doing line work, building toys, then sorting them per area. Everyone happily singing and working! When suddenly one elf put the first toy in a box marked "Salt Lake City" and got surprised it was not empty...
+
+And he got a terrible flashbak, they seemed to have forgotten to deliver 2021 toys in Salt Lake City!
+
+=head2 Go ask Santa
+
+The elves were panicked, hence they reached quickly Santa for a solution. 
+
+He took some time to think about it then had an idea: Let's use the time machine to come back in 2021 and deliver the gifts!
```

Comment:
- ... think about it_,_ then had
- to _go_ back _to_ 2021

Reply:
Edited